src/client/power-pages/actions-hub/TelemetryHelper.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as os from 'os';
+import PacContext from "../../pac/PacContext";
+import CurrentSiteContext from "./CurrentSiteContext";
+import ArtemisContext from '../../ArtemisContext';
+
+const getOperatingSystem = () => {
```

Comment:
@BidishaMS @priyanshu92 aren't we getting at infra level for all the events?

Reply:
I don't think so. These are the common attributes right now: https://github.com/microsoft/powerplatform-vscode/blob/89b06550ad034bf9d79e8170bb0f878e9692a5a3/src/common/OneDSLoggerTelemetry/oneDSLogger.ts#L296C17-L332C59

I don't think we can use `os` package in `common` because it is dependent on node and won't work in Web.

Reply:
I stand corrected. Updated the code to write OS in common properties.